File: python/getParam.py
import sys

def getParam():
    argCount = len(sys.argv) - 1
    if(argCount<2) : return "not enough args, argCount=" + str(argCount) + "<2"
    try:
        a = float(sys.argv[1])
        b = float(sys.argv[2])
        return (a, b)
    except:
        return "NaN"

File: javascript/getParam.js
getParam = function(){
    let argCount = process.argv.length - 2
    if(argCount<2)
      throw new Error("not enough args, argCount=" + argCount + "<2")
    try{
      let a = parseFloat(process.argv[2])
      let b = parseFloat(process.argv[3])
      return {a : a, b : b}
    }
    catch(err){
      throw new Error("NaN")
    }
}

module.exports = {
  getParam: getParam
}
